chain/find.go

```diff
@@ -261,7 +264,7 @@ select height, task
 order by height desc
 ;
 `,
-		len(AllTasks), // arg 0
+		len(indexer.AllTasks),                      // arg 0
```

Comment:
I don't believe this change is going to work in the following scenario:

User runs walks using tasks `messages`, `chaineconomics` and `blocks`. Let's say epochs 4 and 7 produce SKIPs for one of the tasks (say `blocks`) which need to be gapfilled. Then the user `gap finds` over that range for tasks `messages` and `blocks` and not `chaineconomics`. The query would consider the gaps from `blocks` (in epochs 4 and 7) filled because `len(indexer.AllTasks)` is 2 and is the number of records found for heights 4 and 7 is the same count (despite the gaps). (See diagram for vis)

```
 Task State
          ┌────┬────┬────┬────┬────┬────┬────┬────
          │ 2  │ 3  │ 4  │ 5  │ 6  │ 7  │ 8  │ 9
  Blocks  │ ok │ ok │ SK │ ok │ ok │ SK │ ok │ ok
  Messages│ ok │ ok │ ok │ ok │ ok │ ok │ ok │ ok
  Economics ok │ ok │ ok │ ok │ ok │ ok │ ok │ ok
          │    │    │    │    │    │    │    │
          └────┴────┴────┴────┴────┴────┴────┴────
```

This query will probably need to be explicit about which tasks need to match here. I can come back around to help here if needed.

Reply:
This is a mechanical change due to the creation of the indexer package. No logic has changed here.